wallet/src/sender.rs

```diff
@@ -178,16 +178,59 @@ fn build_send_tx(
 		)
 	})?;
 
+	// Get the maximum number of outputs in the wallet
+	let max_outputs =  WalletData::read_wallet(&config.data_file_dir, |wallet_data| {
+		wallet_data.select_coins(
```

Comment:
Not related to this PR (which is good), but it seems `select_coins` could bear some improvement: 6 arguments for a method should probably get refactored in a struct with some sensible default values.

Reply:
Agree, I will look for a way to make in more compact.